app/assets/stylesheets/application.scss

```diff
@@ -21,20 +21,25 @@ $govuk-global-styles: true;
 }
 
 .task-checkboxes {
-  .govuk-checkboxes__item {
-    margin-bottom: govuk-spacing(9);
-  }
-}
-
-.task-checkboxes,
-.clear-legal-documents-task-checkboxes {
   .govuk-checkboxes__item {
     label {
       @extend .govuk-\!-font-weight-bold;
     }
   }
 }
 
+.task-action__wrapper:first-of-type {
+  margin-top: 0;
+}
+
+.task-action__wrapper__padding-normal {
+  margin-top: govuk-spacing(9);
+}
+
+.task-action__wrapper__padding-reduced {
```

Comment:
You can do some Sass magic here:

```css
.task-action__wrapper {
  &__padding-normal {
    margin-top: govuk-spacing(9);
  }
  
  &__padding-normal {
    margin-top: govuk-spacing(2);
  }
}
```

Reply:
Ooh, I thought that only worked for things like identifiers and selectors rather than straight string concatenation, TIL.